### ISSUE TITLE
5.1.0: unexpected token "export" when running jest

### ISSUE THREAD

**Issue by jlowcs** [glennreyes/react-countup#519]
Hi again!

I'm getting another error in my CI since the 5.1.0 release:

![image](https://user-images.githubusercontent.com/3777724/127182251-24de9b39-414d-4f69-8677-a2e15877916a.png)

That error seems to be coming from this:

![image](https://user-images.githubusercontent.com/3777724/127182334-acaba43b-6c45-4c79-9898-bfe8b8cfc181.png)

It appears that the `main` property of `package.json` points towards a ES2015 module instead of a ES5 module?

**Comment by mmarkelov** [comment]
@jlowcs I think that this is coming from [countup issue](https://github.com/inorganik/countUp.js/issues/218).
And I got this error also. So I used this config for jest:
```js
module.exports = {
  ...,
  transformIgnorePatterns: [
    "/node_modules/(?!countup\\.js).+\\.js$",
  ],
};
```

**Comment by jlowcs** [comment]
Oh, that's a good point. I didn't realize that this was coming from countup.js. The issue is old too...

Sorry about that.

**Comment by jlowcs** [comment]
They actually generate an umd module, but the `main` is still pointing on the non-umd module.

**Comment by jlowcs** [comment]
I just created https://github.com/inorganik/countUp.js/issues/262

**Comment by jlowcs** [comment]
And they're gonna make the suggested change, so we won't need that jest hack anymore ;)

**Comment by jlowcs** [comment]
Change was apparently released on countup.js.

**Comment by mmarkelov** [comment]
@jlowcs just checked it with latest version. And it works perfect. I want to remove **warning** package and release new version soon. Thanks!

**Comment by mmarkelov** [comment]
should be fixed in **5.2.0**
